### ISSUE TITLE
Linting LICENSE in a project directory

### ISSUE THREAD

**Issue by geozeke** [DavidAnson/markdownlint#1474]
Thanks for all the hard work on the awesome tool!

This one is oddly specific.

I'm using vscode, and I have a python project with LICENSE file. I added a `.markdownlint.json` file to the project directory to disable `MD033`, which works fine. However, markdownlint is now linting my LICENSE file, suggesting a `MD041` violation. If I delete `.markdownlint.json`, then it stops linting the LICENSE.

Here are the glob patterns set in the vscode plugin:

```text
**/*.{md,mkd,mdwn,mdown,markdown,markdn,mdtxt,mdtext,workbook}
!**/*.code-search
!**/bower_components
!**/node_modules
!**/.git
!**/vendor
```

If I put any other files in the project directory, without a file extension (e.g. TESTFILE), no issues are reported. If I give my LICENSE file a `.txt` extension, no issues are reported. It only happens when I have a `.markdownlint.json` file and a LICENSE file.
